src/cli/commands/files/cat.js

```diff
@@ -31,13 +31,12 @@ module.exports = Command.extend({
         })
         return
       }
-      ipfs.files.cat(path, (err, res) => {
+      ipfs.files.cat(path, (err, file) => {
         if (err) {
           throw (err)
         }
-        res.on('data', (data) => {
-          data.stream.pipe(process.stdout)
-        })
+        console.log(file)
+        file.pipe(process.stdout)
```

Comment:
why console.log and file.pipe(process.stdout?

The test for cat is not checking the output, and that is why this got missed